cmd/root.go

```diff
@@ -91,16 +93,20 @@ Find more information at https://github.com/redhat-developer/odo`,
   # Accessing your Node.js component
   odo url create`,
 	PersistentPreRun: func(cmd *cobra.Command, args []string) {
-
-		// Add extra logging when verbosity is passed
-		if GlobalVerbose {
-			//TODO
-			log.SetLevel(log.DebugLevel)
-		}
-
+		setFlags()
 	},
 }
 
+func setFlags() {
+	flag.Usage = func() {
+		flag.PrintDefaults()
+		os.Exit(2)
+	}
+	flag.CommandLine.Set("logtostderr", "true")
+	flag.CommandLine.Set("v", strconv.Itoa(GlobalVerbose))
```

Comment:
this looks a bit strange, we are setting goflags manualy?

Reply:
I Simply followed: 
https://gist.github.com/heatxsink/7221ebe499b0767d4784

and I observed that glog expects those params on cli so found this as a way to acieve it programmatic-ally.

Reply:
Yes, but in that example he never sets flags "manualy" ,like this.
What about other glog flags like `-log_dir` or `-vmodule`?

Reply:
@anmolbabu Is there a way to allow specification of all glog flags?

Have you tried this approach https://flowerinthenight.com/blog/2017/12/01/golang-cobra-glog ?

Reply:
There is also section in pflags docs that might help: https://github.com/spf13/pflag#supporting-go-flags-when-using-pflag

Reply:
Ack done this time as per the https://github.com/openshift/origin/blob/master/tools/depcheck/pkg/cmd/cmd.go#L29

Reply:
@kadel 
1. GlobalVerbose is not used anywhere now as verbosity setting is 1 time global setting and thereafter each log message carries a verbosity attached inorder for glog to know if the message needs to be logged for the user chosen verbosity..
2. glog doesn't recognize `--verbosity=4` and only understands `-v 4`

Because of 1, I removed GlobalVerbose and let glog handle the verbosity setting directly from cli which means because of 2, `--verbosity=x` can't be used any more only `-v x`  . If we want to preserve even `--verbosity=x` form of flag I'll need to workaround glog by parsing the flag(like we used to do before) and then setting it manually as a flag to glog.

What do you suggest would be good to have?